use/wordcount-example.md

```diff
@@ -0,0 +1,413 @@
+
+Each WordCount example introduces different concepts in the Beam programming model. Begin by understanding Minimal WordCount, the simplest of the examples. Once you feel comfortable with the basic principles in building a pipeline, continue on to learn more concepts in the other examples.
+
+* **Minimal WordCount** demonstrates the basic principles involved in building a pipeline.
+* **WordCount** introduces some of the more common best practices in creating re-usable and maintainable pipelines.
+* **Debugging WordCount** introduces logging and debugging practices.
+* **Windowed WordCount** demonstrates how you can use Beam's programming model to handle both bounded and unbounded datasets. Note: Windowed WordCount is not yet available in the Python SDK.
+
+## MinimalWordCount
+
+Minimal WordCount demonstrates a simple pipeline that can read from a text file, apply transforms to tokenize and count the words, and write the data to an output text file. This example hard-codes the locations for its input and output files and doesn't perform any error checking; it is intended to only show you the "bare bones" of creating a Beam pipeline. In later examples, we will parameterize the pipeline's input and output sources and show other best practices.
+
+To run this example, follow the instructions in the [Beam Examples README](https://github.com/apache/incubator-beam/blob/master/examples/java/README.md#building-and-running). To view the full code, see **[MinimalWordCount](https://github.com/apache/incubator-beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/MinimalWordCount.java).**
+
+**Key Concepts:**
+
+* Creating the Pipeline
+* Applying transforms to the Pipeline
+* Reading input (in this example: reading text files)
+* Applying ParDo transforms
+* Applying SDK-provided transforms (in this example: Count)
+* Writing output (in this example: writing to a text file)
+* Running the Pipeline
+
+The following sections explain these concepts in detail along with excerpts of the relevant code from the Minimal WordCount pipeline.
+
+### Creating the Pipeline
+
+The first step in creating a Beam pipeline is to create a `PipelineOptions object`. This object lets us set various options for our pipeline, such as the pipeline runner that will execute our pipeline and any runner-specific configuration required by the chosedn runner. In this example we set these options programmatically, but more often command-line arguments are used to set `PipelineOptions`. 
+
+In our example, we selected `DataflowRunner` for our Beam runner. This means our pipeline will be executed in the cloud using Google Cloud's Dataflow Service. There are other runners you can specify for executing your pipeline, such as `FlinkRunner` or `SparkRunner`. If you omit specifying a runner, your pipeline will be executed locally. 
```

Comment:
Why are we suggestion DataflowRunner? Let's just leave this as defaulting to direct and offering options to swap to other runners. 

Reiterate comment from above that these code changes will be parameterized away in the next section.